.travis.yml

```diff
@@ -31,25 +31,23 @@ install:
 
 script:
   # Create a test environment
-  - conda create --quiet --yes -n test python=$python
+  - conda create --quiet --yes -n test python=$python conda-build conda-verify jinja2 pip nose nose-timer coverage modeller
   # Activate the test environment
   - source activate test
-  # Install extras for OpenEye, Modeller and testing
-  - conda install --yes --quiet pip nose nose-timer coverage modeller
   # Install OpenEye toolkit
   - pip install $OPENEYE_CHANNEL openeye-toolkits && python -c "import openeye; print(openeye.__version__)"
   # Build the recipe
   - conda build devtools/conda-recipe
   # Install the package
-  - conda install --quiet --yes --use-local ${PACKAGENAME}-dev
+  - conda install -c $HOME/miniconda/envs/test/conda-bld/ --quiet --yes --use-local ${PACKAGENAME}-dev
   # Test the package
-  - cd devtools && nosetests $PACKAGENAME --with-coverage --nocapture --verbosity=2 --with-timer -a '!slow' && cd .. && if [[ $CONDA_PY = 3.5 ]]; then devtools/travis-ci/build_docs.sh; fi
+  - cd devtools && nosetests $PACKAGENAME --with-coverage --nocapture --verbosity=2 --with-timer -a '!slow' && cd .. && if [[ $CONDA_PY = 3.7 ]]; then devtools/travis-ci/build_docs.sh; fi
 
 env:
   matrix:
-    - python=3.5 CONDA_PY=3.5
-    - python=3.5 CONDA_PY=35 DEVOMNIA=true
     - python=3.6 CONDA_PY=3.6
+    - python=3.7 CONDA_PY=3.7
+    - python=3.7 CONDA_PY=3.7 DEVOMNIA=true
```

Comment:
Whoops! In the end, I didn't post the comment with the review.

I think the `CONDA_PY` here must be set to the notation `3.7` to make this command work: https://github.com/choderalab/yank/blob/master/devtools/travis-ci/build_docs.sh#L7